### DISCUSSION TITLE
BUG channels 影响django 正常的http并发

### DISCUSSION THREAD

**Discussion by wuqiang-001** [django/channels#1827] [Help!]
我安装好 channels 和 django  可以正常工作

在我开启多线程并发django http请求时  出现工作不能正常接收返回
![image](https://user-images.githubusercontent.com/85499933/158211684-3729a57f-30bc-4e20-aab5-18be9aa8ab6d.png)
![image](https://user-images.githubusercontent.com/85499933/158211967-649a9b69-fd0f-4ccc-be34-380b336acfaf.png)

![image](https://user-images.githubusercontent.com/85499933/158212289-239e8d2e-f53d-4931-9b3c-cf8a4d7dac01.png)

如果我描述的不清楚  ,请及时与我联系

**Comment by carltongibson** [comment]
Hi, sorry — there isn't really enough to go on here. I can't see what the problem would be from the information you've given.

**Reply by wuqiang-001** [reply]
问题已经得到解决，谢谢您的答复。
